### ISSUE TITLE
[Test][A11y]: ACOM | Service Page [Screen Readers]: NVDA announces same information twice after activating "What is basic?" button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15126]
### Steps to reproduce:
1. Enable NVDA and open above URL in "Chrome" browser.
2. "Azure Active Directory | Microsoft Azure" page will open.
3, Navigate to "What is basic?"  button using "Tab" key-> activate it->popup will appear.
4. Observe that NVDA announces same information twice after activating "What is basic?" button.
### Actual behavior:
NVDA announces same information twice after activating "What is basic?" button.

NVDA announces as:  "What is basic?  dialog Close basic popover button What is basic? button  Close basic popover The basic configuration was selected based on the most common, low-price product settings that customers choose. Region was chosen based on greatest availability. The intent is to show a typical, easily implemented low price point that has high availability."
### Expected behavior:
NVDA shouldn't announce same information twice after activating "What is basic?" button.

NVDA should announce as:  "What is basic?  dialog Close basic popover button The basic configuration was selected based on the most common, low-price product settings that customers choose. Region was chosen based on greatest availability. The intent is to show a typical, easily implemented low price point that has high availability. "
### NVDA logs, crash dumps and other attachments:
Same issue replicates for "What is most popular?", and "What is max performance?" buttons.
#### Windows version: Version 22H2
OS build 22621.1848
Experience: Windows Feature Experience Pack 1000.22642.1000.0

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

To be able to reproduce this issue, we need minimal a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

Kind Regards,
NV Access Software Developers

**Comment by msftedad** [comment]
Hi Seanbudd ,this issue is reproducible in public version also, still we are following up with the engineering team regarding the code pen as will make this available to you soon. Thanks
